inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/config/model/instrumentation/scope/AdvancedScopeSettings.java

```diff
@@ -24,4 +24,8 @@
      */
     private boolean disableSafetyMechanisms = false;
 
+    /**
+     * Defines whether lambda classes should be excluded from the declared scope.
+     */
+    private boolean excludeLambdas = true;
```

Comment:
I would rather put in on the instrumentation level, next to "ignore-packages" I would add an "ignore-lambdas",
Otherwise it would be impossible to exclude lambdas being instrumented with special sensors.
Then you can also jsut keep your matchers as they are and add a corresponding check to InstrumentationConfigurationResolver.isIgnoredClass()